ext/src/main/java/dev/brahmkshatriya/echo/extension/spotify/Authentication.kt

```diff
@@ -46,6 +45,22 @@ class Authentication(
         return accessToken!!
     }
 
+    private fun generateTotp(time: Long): String {
+        val secretCipherBytes = listOf(
+            12, 56, 76, 33, 88, 44, 88, 33, 78, 78, 11, 66, 22, 22, 55, 69, 54
+        ).mapIndexed { index, byte -> byte xor (index % 33 + 9) }
+            .joinToString("") { "%02x".format(it) }
+
+        val secret =
```

Comment:
Could you move this to companion object instead?

Reply:
No need to calculate this everytime, we need to generate the totp

Reply:
This way it would be easier to change out the secret imo, and if the compiler is any good this should be calculated at compiletime

Reply:
What I meant was to put line 49-55 in a companion object, I dont think the compiler is smart enough to figure out it should recalculate this

Reply:
putting that lines 49-55 to companion object, shouldnt affect the ease of the changing the keys